### PR TITLE
Setup factory_girl.

### PR DESCRIPTION
**Pull request by dpdawson**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -45,6 +45,8 @@ group :development, :test do
   gem 'rubocop'
   gem 'dotenv-rails'
 
+  gem 'factory_girl_rails', '~> 4.4.1'
```
Comment (schneidmaster) [inline Gemfile]: Is there a reason why factory_girl is needed in the development environment in addition to test?

File: spec/support/factory_girl.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
+
+  config.before(:suite) do
+    begin
+      DatabaseCleaner.start
```
Comment (schneidmaster) [inline spec/support/factory_girl.rb]: This code is somewhat redundant with [these lines](https://github.com/dpdawson/ArborApp/blob/feature/setup-factory-girl/spec/rails_helper.rb#L67-L78) from the rails_helper - I would recommend combining them (probably moving the DatabaseCleaner calls from the rails_helper into this file)

File: Gemfile (hunk 1 above)
Comment (dpdawson) [inline Gemfile]: I think you have to add it to development for rails generate factories instead of fixtures.

Comment (schneidmaster) [inline Gemfile]: Ah, makes sense.

**Pull request merged**